src/Autohide.js

```diff
@@ -182,10 +186,51 @@ define(function (require, exports, module) {
         exposeDelay = delay;
     }
 
+    function showDelayDialog() {
+        var Dialog = Dialogs.showModalDialog(
+            brackets.DIALOG_ID_SAVE_CLOSE,
+            Strings.AUTOHIDE_DELAY_MODAL_TITLE,
+            Mustache.render(sliderTemplate, {
+                exposeDelay: exposeDelay + Strings.AUTOHIDE_DELAY_MODAL_MS_LABEL
```

Comment:
This doesn't seem to work for setting the current slider value.

Reply:
Ups! I wanted to go too fast . It worked until I added the string ... hehe